packages/suite/src/components/suite/labeling/MetadataLabeling/MetadataLabeling.tsx

```diff
@@ -411,7 +411,10 @@ export const MetadataLabeling = ({
         showActionButton && (!payload.value || (payload.value && pending));
 
     return (
-        <LabelContainer data-testid={labelContainerDataTest} onClick={e => e.stopPropagation()}>
+        <LabelContainer
+            data-testid={labelContainerDataTest}
+            onClick={e => editActive && e.stopPropagation()}
```

Comment:
I just looked at this PR that introduced this `stopPropagation`. https://github.com/trezor/trezor-suite/pull/16882 I would compare the original goal of the fix with the current state. Because this looks weird. Maybe there is a way to delete both.

But up to you

Reply:
There was an issue with coin control. Now it should be also fixed.